R/ccfr_uncertainty.R

```diff
@@ -0,0 +1,74 @@
+#' Corrected case fatality rate
```

Comment:
ccfr_estimation might be better name, as this is what function does. But not essential.

Reply:
Would suggest `estimate_ccfr` if renaming to follow function-as-verb convention.

Reply:
Renamed throughout to `estimate_ccfr`, primarily in https://github.com/epiverse-trace/datadelay/pull/11/commits/74af9360e3c410d63b971c81235547872df95f53